Makefile

```diff
@@ -9,8 +24,47 @@ GO_FILES ?= $$(find . -name '*.go' | grep -v generated)
 	@./.dapper.tmp -v
 	@mv .dapper.tmp .dapper
 
+.nodapper:
+	$(info Checking essential build tools.)
+	@if [ ! -d $(WD)/bin ] ; then \
+		mkdir $(WD)/bin ; \
+	fi
+	$(info Checking go version for compatibility.)
+	@if [ ! -d $(GOROOT) ] ; then \
+		echo "No go found, fetching compatible version." ; curl -sL https://go.dev/dl/go$(GO_VERSION).linux-$(ARCH).tar.gz | tar -C $$PWD/bin -zxf - ; \
```

Comment:
we shouldn't do this either.
we can pull golang from backports: https://packages.debian.org/bookworm-backports/golang